supervisory-authorities/derfmdr.json

```diff
@@ -0,0 +1,16 @@
+{
+    "slug": "derfmdr",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Rundfunkbeauftragter für den Datenschutz beim MDR",
+    "address": "Kantstraße 71-73\n04275 Leipzig\nDeutschland",
+    "phone": "+49 341 3000",
```

Comment:
https://www.rundfunkdatenschutzkonferenz.de/kontakt has better phone and fax numbers.